src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -466,13 +466,35 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         return emails;
     }
 
+    /**
+     * Generates the feedback session closed email for both
+     * the instructors and the students with the given
+     * {@code userName} and {@code userEmail}.
+     */
     private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
             FeedbackSessionAttributes session, String userName, String userEmail) {
+
```

Comment:
Although it is good to be consistent in the code style within the class, conserving the change history is usually a higher priority. This makes it easier for future developers to trace the history of the changes and understand the context in which the code was originally written.